theorems/T000630.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: T000630
+if:
+  and:
+  - P000003 : true
+  - P000030: true
+  - P000029: true
+then:
+  P000018: true
+refs:
+- mathse: 190078
+  name: Every paracompact space with the Suslin property is Lindelöf
+- mathse: 1301345
+  name: How to prove that every Paracompact space with the Suslin property is Lindelof
+---
+
+See {{mathse:190078}} and {{mathse:1301345}}.
```

Comment:
```suggestion
---
uid: T000630
if:
  and:
  - P000003 : true
  - P000030: true
  - P000029: true
then:
  P000018: true
refs:
- mathse: 5004462
  name: Is every paracompact CCC space Lindelöf?
---

See {{mathse:5004462}}.
```

We can update this theorem in a separate PR to use para-Lindelof (when someone wants to add para-Lindelof).

Of course this suggestion assumes my proof at Math SE is valid, so wait until it's vetted better before taking this suggestion.

Reply:
I think @david20000813 found an error, which I think mirrors @prabau's concern with Dan Ma's blog post.

So this suggestion is certainly premature.

Reply:
The Math SE post now has a valid proof, so this suggestion may now be considered.